cvat-ui/src/components/annotation-page/standard-workspace/styles.scss

```diff
@@ -119,3 +119,77 @@
         margin: 0px 5px;
     }
 }
+
+.cvat-canvas-context-menu {
+    opacity: 0.6;
+    position: fixed;
+    width: 300px;
+    z-index: 10;
+    max-height: 50%;
+    overflow-y: auto;
+
+    &:hover {
+        opacity: 1;
+    }
+}
```

Comment:
Why have you duplicated style from ``src/components/annotation-page/styles.scss``?